### PR TITLE
Update Ukrainian translation for security.md

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
Refs: https://github.com/nodejs/nodejs.org/pull/2179

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-uk: Would you mind reviewing it? Thanks!

**Review by listochkin — APPROVED** [review]
Thank you so much!

File: locale/uk/security.md [hunk 1]
```diff
@@ -33,16 +40,14 @@ title: Безпека
...
 
-
 ## Отримання оновлень безпеки
 
 Сповіщення, що стосуються безпеки, будуть поширюватись за допомогою:
 
-- [https://groups.google.com/group/nodejs-sec](https://groups.google.com/group/nodejs-sec)
-- [https://nodejs.org/en/blog](https://nodejs.org/en/blog)
-
+- <https://groups.google.com/group/nodejs-sec>
```
Comment (sudodoki) [inline locale/uk/security.md:47]: will this produce correct links in html?

Comment (vsemozhetbyt) [reply locale/uk/security.md]: I've copied from the English source.

**Review by ghaiklor — APPROVED** [review]

File: locale/uk/security.md (hunk 1 above)
Comment (vsemozhetbyt) [reply locale/uk/security.md:47]: As per the [the English source](https://nodejs.org/en/security/), yes, it seems so.

**Review by willin — APPROVED** [review]

**Pull request merged**